components/boards/boards.c

```diff
@@ -177,4 +216,33 @@ void board_led_state(uint32_t state)
     break;
   }
   #endif
-}
+
```

Comment:
this code is exactly the same as above switch, can you merge them together you can try to `define rgb_set()` that expand to neopixel_set or setAPA depending on the PIN_xxx.

Reply:
Sure, I was thinking of that when I implemented it, but wasn't sure if it was ok to change the neopixel stuff as that wasn't my code ;)